data/campaigns/World_Conquest/lua/map/scenario_utils/plot.lua

```diff
@@ -45,29 +45,29 @@ function add_plot(scenario, scenario_num, nplayers)
 	elseif scenario_num == 2 then
 		local r = wesnoth.random(nplayers)
 		start_message(r, true, _ "Ahhh. Just look at these fertile lands, ripe for conquest!")
-		start_message("4", true, _ "What's this?! Foreign invaders have set foot upon our shores!")
-		start_message("5", true, _ "We'll send them back to where they came from quick enough. All troops, to me!")
+		start_message("4", true, _ "What’s this?! Foreign invaders have set foot upon our shores!")
+		start_message("5", true, _ "We’ll send them back to where they came from quick enough. All troops, to me!")
 
 		end_message("1,2,3", false, _ "We win! Truly ours will be a glorious empire!")
 		end_message("1,2,3", true, _ "Do not get hasty! To reach the great continents of the east we must subdue the savage coast of Moragdu, notorious for its corsairs, before we can advance further.")
 	elseif scenario_num == 3 then
-		start_message("4", true, _ "What's this? It is a long time since strangers have set foot upon Moragdu--and in such great numbers too!")
+		start_message("4", true, _ "What’s this? It is a long time since strangers have set foot upon Moragdu--and in such great numbers too!")
 		start_message("5", true, _ "The time of prophecies is upon us! The destroyers have arrived from beyond the sea!")
-		start_message("6", true, _ "I don't know what the lizard is babbling about, but we'll sort these 'destroyers' out quickly enough.")
+		start_message("6", true, _ "I don’t know what the lizard is babbling about, but we’ll sort these ‘destroyers’ out quickly enough.")
 
 		end_message("1,2,3", true, _ "Victory is ours! Let us take sail in search of new lands to conquer!")
```

Comment:
Should "take sail" be "set sail"?

Reply:
I assumed it was a nautical term, like 'take in the sails', but yes, probably makes more sense to be set sail.